client/gutenberg/editor/calypsoify-iframe.tsx

```diff
@@ -417,8 +417,8 @@ class CalypsoifyIframe extends Component<
 					post: translate( 'Add new post' ),
 				},
 				listHeadings: {
-					page: translate( 'Recent Pages' ),
-					post: translate( 'Recent Posts' ),
+					page: translate( 'Recent pages' ),
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.